tests/5.0/target/test_target_default.c

```diff
@@ -0,0 +1,42 @@
+//===------ test_target_default.c --------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// Tests the target directive with parallel for + default clause.
+//
+//===------------------------------------------------------------------------===//
+
+#include <stdio.h>
+#include "omp.h"
+#include "ompvv.h"
+
+#define N 100
+
+
+int IfTstPassed = 1; // 1 is passed 0 is failed
+
+int main(int argc, char** argv) {
+  int count = 123, errors = 0;
```

Comment:
Add "OMPVV_TEST_OFFLOADING;" --> This helps with the makefile harness to determine where the test was executed.

Reply:
Added "OMPVV_TEST_OFFLOADING;" to all the target related tests...